# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1323249352 18000
#      Wed Dec 07 04:15:52 2011 -0500
# Node ID 82f5f76726a20cbc18ee9ce7b1577b5d7aef11ba
# Parent  5f0bb45e615c0732adcda1a420c30b85d265403c
don't delete FLTK plot window widgets

* __init_fltk__.cc (plot_window::plot_window): Don't delete canvas,
status, or uimenu.

diff --git a/src/DLD-FUNCTIONS/__init_fltk__.cc b/src/DLD-FUNCTIONS/__init_fltk__.cc
--- a/src/DLD-FUNCTIONS/__init_fltk__.cc
+++ b/src/DLD-FUNCTIONS/__init_fltk__.cc
@@ -744,19 +744,16 @@ public:
   }
 
   ~plot_window (void)
   {
     canvas->hide ();
     status->hide ();
     uimenu->hide ();
     this->hide ();
-    delete canvas;
-    delete status;
-    delete uimenu;
   }
 
   double number (void) { return fp.get___myhandle__ ().value (); }
 
   void renumber (double new_number)
   {
     if (canvas)
       {
